.circleci/config.yml

```diff
@@ -1,36 +1,70 @@
 # Use the latest 2.1 version of CircleCI pipeline process engine. See: https://circleci.com/docs/2.0/configuration-reference
 version: 2.1
 
-orbs:
-  golangci-lint: timakin/golangci-lint@volatile
-
 workflows:
   build:
     jobs:
-      - golangci-lint/lint
-      - test
+      - lint:
+          dir: ./promql
+      - test:
+          dir: ./promql
+      - lint:
+          dir: ./remote_write
+      - test:
+          dir: ./remote_write
 
 jobs:
   test:
+    parameters:
+      dir:
+        type: string
     working_directory: ~/repo
     docker:
       - image: circleci/golang:1.16.0
```

Comment:
This image is depreciated, you should use cimg

Reply:
Whats cimg?

Reply:
I guess https://circleci.com/developer/images/image/cimg/go